js/main.js

```diff
@@ -18,13 +18,27 @@ import './src/eclipseApi'
 import 'jquery-parallax.js/parallax.min.js'
 import './src/parallax'
 
+const solsticeSliderHome = () => {
+    var owl = $('.solstice-slider-home');
+    owl.owlCarousel({
+        items:1,
+        autoplay:true,
+        autoplayTimeout:6000,
+        autoplayHoverPause:true,
+        autoplaySpeed: 2000,
+        loop:true,
+    });
+}
 
-var owl = $('.solstice-slider-home');
-owl.owlCarousel({
-    items:1,
-    autoplay:true,
-    autoplayTimeout:6000,
-    autoplayHoverPause:true,
-    autoplaySpeed: 2000,
-    loop:true,
+$(window).on("load", function() {
+    solsticeSliderHome();
+});
+
+
+
+$("body").on("shown.ef.featured_story", function(e) {
+    var owl = $('.solstice-slider-home');
+    owl.trigger('destroy.owl.carousel');
```

Comment:
Why do you trigger a destroy and refresh event.  I feel like we are doing something wrong here. I don't really understand.

Adding comments in your code would help us understand if this is really needed.

Reply:
Ok i'll add some comments in there.

The destroy and refresh here helps because owl carousel needs to be updated after the items have been added to the container div with ajax.

Reply:
How about you try to load the data before initializing owl?

Reply:
That would make more sense, I'll try that.